src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -90,6 +90,21 @@ typedef enum HorseType {
     /* 1 */ HORSE_HNI
 } HorseType;
 
+typedef enum HorseParamType {
+    /* 00 */ HORSE_PTYPE_0,
```

Comment:
we generally just say "type", I dont think theres a need to say "param type" or to put P in the name

Reply:
The issue is that there is already a `HorseType` enum just above this one that arguably represents what type of horse it is

Reply:
then what is a "param type" and how does it differ from the actual type? surely we can come up with something a bit more descriptive here

Reply:
`HorseParamType` is simply what is stored in actor `EnHorse`'s `actor.params` after `actor.params` has been sanitized a bit in EnHorse_Init. More specifically, it's after the bit storing `HorseType` is stripped out, and a params of all 1's is converted into `HORSE_PTYPE_1`.

Since my primary goal with this PR is to document `z_horse.c` and not `z_en_horse.c`, I do not know what all of the values fully represent. Many of them are initialization related, but there's some complexity with `EnHorse_Freeze`/`EnHorse_Frozen` that requires digging in deeper than I feel like right now.

Reply:
ok. if the understanding is that this will change later im cool with it.